airtel/client.go

```diff
@@ -117,7 +117,7 @@ func NewDefault(pin, clientId, secret, grant string) (*Client, error) {
 
 // do wraps the Client.Do function by creating the Request and
 // unmarshalling the response according to user expected output.
-func (c *Client) do(ctx context.Context, method, path string, in, out interface{}, headers http.Header) (*client.Response, error) {
+func (c *Client) do(ctx context.Context, method, path string, in, out interface{}, headers http.Header, addToken bool) (*client.Response, error) {
```

Comment:
`addToken` should something like `authRequired` gives the right message, I hope that what it means and I'm not mistaken.

Keep up the good work

Reply:
sure your are right.
Thank